src/Raven.Server/Documents/Replication/ConflictManager.cs

```diff
@@ -273,12 +273,20 @@ private static void InvalidConflictWhenThereIsNone(string id)
                 existingTombstone.ChangeVector = ChangeVectorUtils.MergeVectors(incomingChangeVector, existingTombstone.ChangeVector);
                 using (Slice.External(context.Allocator, existingTombstone.LowerId, out Slice lowerId))
                 {
-                    _database.DocumentsStorage.ConflictsStorage.DeleteConflicts(context, lowerId, null, existingTombstone.ChangeVector);
+                    _database.DocumentsStorage.ConflictsStorage.DeleteConflicts(context, lowerId, null, context.GetChangeVector(existingTombstone.ChangeVector));
                 }
                 return true;
             }
 
             return false;
         }
+
+        
+        public static int Compare(DocumentConflict x, DocumentConflict y, IChangeVectorOperationContext context)
+        {
+            var cvx = new ChangeVector(x.ChangeVector, context);
+            var cvy = new ChangeVector(y.ChangeVector, context);
+            return string.Compare(cvx.Version.AsString(), cvy.Version.AsString(), StringComparison.Ordinal);
```

Comment:
you are getting 2 strings, converting them to change vectors and then converting them to string?

Reply:
`AsString` isn't allocating a new string.

Reply:
```
cvx.CompareVersion(xvy);
```